doc/source/glossary.rst

```diff
@@ -2,10 +2,106 @@
 Glossary
 ********
 
-.. toctree::
-
 .. glossary::
 
+    (`n`,)
+
+      A tuple with one element. The trailing comma distinguishes a one-element
+      tuple from a parenthesized ``n``.
+
+
+    -1
+
+      Used as a dimension entry, ``-1`` instructs NumPy to choose the length
+      that will keep the total number of elements the same.
+
+
+    ``...``
```

Comment:
also know as an [ellipsis](https://docs.python.org/3/library/constants.html#Ellipsis)

Reply:
Might want to add a link to ``:data:`python:Ellipsis` `` here, just as is done for ``:``.